.github/workflows/check.yaml

```diff
@@ -82,7 +82,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [[ubuntu-22.04], [Ubuntu_ARM64_4C_16G_01]]
+        runs-on: [[ubuntu-22.04], [ubuntu-24.04], [Ubuntu_ARM64_4C_16G_01]]
```

Comment:
This probably needs an **update** rather than **add**. e.g. `ubuntu-22.04` -> `ubuntu-24.04` becuase we only care about the difference in arch